hardhat.config.ts

```diff
@@ -17,13 +17,26 @@ dotenv.config();
 
 const config: HardhatUserConfig = {
   solidity: {
-    version: '0.8.13',
-    settings: {
-      optimizer: {
-        enabled: true,
-        runs: 200,
+    compilers: [
+      {
+        version: '0.8.9',
```

Comment:
Is this for Streamr?

Reply:
It was at first when I was using a mock. We don't need anymore. Thanks for pointing.